src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -322,7 +332,7 @@ const ChatInfoScreen = () => {
         selectedUser={selectedUser}
         isOpen={openModal}
         onPress={handlePressPopup}
-        name={selectedUser?.user?.username || selectedUser?.user?.name || selectedUser?.username}
+        name={getUsername(selectedUser)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in this line is replacing the `name` prop value with the result of the `getUsername` function. This change is consistent with the new logic introduced for handling anonymous usernames.
```diff
- name={selectedUser?.user?.username || selectedUser?.user?.name || selectedUser?.username}
+ name={getUsername(selectedUser)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->